database/migrations/2019_09_28_171753_config_fix.php

```diff
@@ -0,0 +1,577 @@
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'full_photo',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'delete_imported',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'Mod_Frame',
+				'value' => '1',
+				'cat' => 'Mod Frame',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'Mod_Frame_refresh',
+				'value' => '30000',
+				'cat' => 'Mod Frame',
+				'type_range' => INT,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'image_overlay_type',
+				'value' => 'desc',
+				'cat' => 'Gallery',
+				'type_range' => 'exif|desc|takedate',
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'compression_quality',
+				'value' => '90',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_page_enable',
+				'value' => '1',
+				'cat' => 'Mod Welcome',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'landing_owner',
+				'value' => 'John Smith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_title',
+				'value' => 'John Smith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_subtitle',
+				'value' => 'Cat, Dogs & Humans Photography',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_facebook',
+				'value' => 'https://www.facebook.com/JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_flickr',
+				'value' => 'https://www.flickr.com//JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_twitter',
+				'value' => 'https://www.twitter.com//JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_instagram',
+				'value' => 'https://instagram.com/JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_youtube',
+				'value' => 'https://www.youtube.com//JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_background',
+				'value' => 'dist/cat.jpg',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'thumb_2x',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'small_2x',
+				'value' => '1',
```

Comment:
This defaulted to `0` until now, I believe. Personally I always set it to `1` in my installations but I want to be sure that this is an intentional change. It does, after all, come with additional processing overheads and increased space and bandwidth requirements...

Reply:
My gut says default to on (for both of these), but worth raising.